Unit flake in k8s.io/kubernetes/contrib/mesos/pkg/queue

From:
https://pantheon.corp.google.com/storage/browser/kubernetes-jenkins/pr-logs/pull/26389/kubernetes-pull-test-unit-integration/28888/

```
ok  	k8s.io/kubernetes/contrib/mesos/pkg/podutil	1.436s
ok  	k8s.io/kubernetes/contrib/mesos/pkg/proc	4.400s
FAIL	k8s.io/kubernetes/contrib/mesos/pkg/queue	3.608s
ok  	k8s.io/kubernetes/contrib/mesos/pkg/redirfd	1.076s
```

No additional info (not sure why?)

In the future, would you mind posting public links? 😄 

From https://console.cloud.google.com/m/cloudstorage/b/kubernetes-jenkins/o/pr-logs/pull/26389/kubernetes-pull-test-unit-integration/28888/artifacts/junit_v1%2Cextensions-v1beta1%2Cfederation-v1alpha1_20160602-101104.stdout
```
--- FAIL: TestDQ_ordered_add_pop (3.00s)
	delay_test.go:211: totalDelay < 3s: 2.889987265s
```

--- a/contrib/mesos/pkg/queue/delay_test.go
+++ b/contrib/mesos/pkg/queue/delay_test.go
@@ -180,12 +180,13 @@ func TestDQ_ordered_add_pop(t *testing.T) {
 	t.Parallel()
 
 	dq := NewDelayQueue()
+
+	before := time.Now()
 	dq.Add(&testjob{d: 2 * time.Second})
 	dq.Add(&testjob{d: 1 * time.Second})
 	dq.Add(&testjob{d: 3 * time.Second})
 
 	var finished [3]*testjob
-	before := time.Now()
 	idx := int32(-1)
 	ch := make(chan bool, 3)
 	//TODO: replace with `for range finished` once Go 1.3 support is dropped

